### PR TITLE
Fixed spelling

### PR DESCRIPTION
**Pull request by vishalsodani**

### CODE REVIEW

File: boa/examples/classes.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ use boa::{
 // We derive `Debug`, `Trace` and `Finalize`, It automatically implements `NativeObject`
 // so we can pass it an object in JavaScript.
 //
-// The fields of the sturct are not accesable by JavaScript unless accessors are created for them.
+// The fields of the sturct are not accessible by JavaScript unless accessors are created for them.
```
Comment (tofpie) [inline boa/examples/classes.rs:13]:
```suggestion
// The fields of the struct are not accessible by JavaScript unless accessors are created for them.
```

File: boa/examples/classes.rs [hunk 2]
```diff
@@ -58,20 +58,20 @@ impl Class for Person {
 
     // This is what is called when we do `new Person()`
     fn constructor(_this: &Value, args: &[Value], context: &mut Context) -> Result<Self> {
-        // we get the first arguemnt of undefined if the first one is unavalable and call `to_string`.
+        // we get the first argument of undefined if the first one is unavailable and call `to_string`.
```
Comment (tofpie) [inline boa/examples/classes.rs:61]:
```suggestion
        // we get the first argument or undefined if the first one is unavailable and call `to_string`.
```

**Review by tofpie — CHANGES_REQUESTED** [review]

**Review by tofpie — APPROVED** [review]
This looks good to me! Thanks!

**Review by Razican — APPROVED** [review]
Thanks for the fixes!

**Review by RageKnify — APPROVED** [review]

File: boa/examples/classes.rs [hunk 3]
```diff
@@ -58,20 +58,20 @@ impl Class for Person {
...
+        // we get the first argument or undefined if the first one is unavailable and call `to_string`.
         //
         // This is equivalent to `String(arg)`.
         let name = args
             .get(0)
             .cloned()
             .unwrap_or_default()
             .to_string(context)?;
-        // we get the second arguemnt of undefined if the first one is unavalable and call `to_u32`.
+        // we get the second argument of undefined if the first one is unavailable and call `to_u32`.
```
Comment (RageKnify) [inline boa/examples/classes.rs:69]:
```suggestion
        // we get the second argument or undefined if it is unavailable and call `to_u32`.
```

File: boa/examples/classes.rs [hunk 4]
```diff
@@ -58,20 +58,20 @@ impl Class for Person {
 
     // This is what is called when we do `new Person()`
     fn constructor(_this: &Value, args: &[Value], context: &mut Context) -> Result<Self> {
-        // we get the first arguemnt of undefined if the first one is unavalable and call `to_string`.
+        // we get the first argument or undefined if the first one is unavailable and call `to_string`.
```
Comment (RageKnify) [inline boa/examples/classes.rs:61]:
```suggestion
        // we get the first argument or undefined if it is unavailable and call `to_string`.
```

**Review by RageKnify — CHANGES_REQUESTED** [review]
I think my proposed changes make it more understandable.
With one in line 69 including an `of` that should be `or`.

File: boa/examples/classes.rs (hunk 4 above)
Comment (vishalsodani) [reply boa/examples/classes.rs:61]: My suggestion:

We get the first argument. If it is unavailable we get `undefined`. And, then call `to_string()`.

Comment (RageKnify) [reply boa/examples/classes.rs:61]: Sure, you can use the same sentence structure for the other comment. :+1:

**Review by RageKnify — APPROVED** [review]

**Pull request merged**